lots of updates (#16)
nmfs-openscapes/nmfs-openscapes.github.io: champions.qmd

-   **Who:** NOAA Fisheries staff, affiliates, contractors, and others who are interested in improving their data science workflows. We aim to remove barriers to outside collaboration. It is important that those who participate can do so in all or most sessions, and are energized to be interacting with like-minded individuals across NOAA Fisheries. Sign up to participate individually, with a colleague or your team.

### Champions Cohort Timeline

This image gives a visual sense of time span, cadence, and time commitments for an Openscapes Cohort

![](images/openscapes_cohort-timeline.png)

## Sign up for Fall 2025

Sign up with a colleague or team by Sept 8, 2025 by 10am PT via [this Google Spreadsheet](https://docs.google.com/spreadsheets/d/1fpMJpReMuAW3W_U7rGSBkM1BgzWdVySLqLec-xoZYUc/). Please indicate the Cohort date-times you are available; we will confirm participation via email by Sept 15.

## Attend an "Ask Us Anything" info session

> Eli Holmes and Julie Lowndes will host "Ask Us Anything" info sessions on Sept 5 and 11 (details in table below).

Reviewer: ```suggestion
NOAA Fisheries Openscapes will host "Ask Us Anything" info sessions on Sept 2 & 3 (details in table below).
```

**Consider giving a talk at your Center/Regional Office** (talk/brown-bag, 5 mins at All Hands…). Please add sessions to [Sign up sheet, 'Talks, AMA Schedule'](https://docs.google.com/spreadsheets/d/10zTz6i-ZuwnQhCTG63jRku0BtVwgc9e1tqnfGuiyM2c/edit?gid=1423956583#gid=1423956583) tab and to the NMFS Open Science Google Calendar so others can find them.

Here are [slides](https://docs.google.com/presentation/d/1P9pDjtSxVy74HSMwGPvNA97d7tG8zbDQJSInsNTC1VM/edit?slide=id.g2f66fdc8be7_1_146#slide=id.g2f66fdc8be7_1_146) introducing the Fall 2025 Champions Program with background information we can all use to present.

**Table.** Center / Office info sessions.

To join an info session below, find Google Meet links in this [Google Sheet](https://docs.google.com/spreadsheets/d/1fpMJpReMuAW3W_U7rGSBkM1BgzWdVySLqLec-xoZYUc/edit?gid=1423956583#gid=1423956583) or in the NMFS Open Science Google Calendar.

|  |  |  |  |  |  |
|-----------|-----------|-----------|-----------|-----------|-----------|
| **Date** | **Time** | **Center/Office** | **Event info name / length** | **Mentor/Point person** | **Speaker** |  |

> | 2025-09-05 | 11:00 am PT | All NMFS | Openscapes Champions info Q&A / 50 min | Eli Holmes | Eli Holmes, Julie Lowndes |

Reviewer: ```suggestion
| 2025-09-02 | 12:00 pm PT | All NMFS | Openscapes Champions info Q&A / 50 min | Julie Lowndes | Julie Lowndes + TBD |
```

> | 2025-09-11 | 12:00 pm PT | All NMFS | Openscapes Champions info Q&A / 50 min | Eli Holmes | Eli Holmes, Julie Lowndes |

Reviewer: ```suggestion
| 2025-09-03 | 12:00 pm PT | All NMFS | Openscapes Champions info Q&A / 50 min | Julie Lowndes | Julie Lowndes + TBD |
```

**Seaside Chats are the way for people outside the Cohort to participate.** Part of the Champions program design is that participants have “Seaside Chats” in the off-weeks during the 2 months, where they focus on applying what they learned into their own work and include other colleagues or team members. In many cases, not all team members who signed up will be selected to participate in the 5 Cohort Calls. These will be organized and led by the Champions using guidance from Openscapes, and there is no “prework” to attend.

> ### Decision criteria for 2024 Program

Reviewer: ```suggestion
### Decision criteria for 2024 Program

*We will reuse and build on this criteria for the 2025 program.* 
```
